src-tauri/src/commands.rs

```diff
@@ -21,3 +21,23 @@ pub async fn notes_show(app_handle: tauri::AppHandle, id: u32) -> Result<Note, S
         Err(_) => Err("note not found".to_string()),
     }
 }
+
+#[tauri::command]
+pub async fn user_index(app_handle: tauri::AppHandle) -> Result<Vec<User>, String> {
+    let state: State<'_, AppState> = app_handle.state();
+
+    match state.user_manager().index().await {
+        Ok(user) => Ok(user),
+        Err(_) => Err("failed to list users".to_string()),
+    }
+}
+
+#[tauri::command]
+pub async fn user_show(app_handle: tauri::AppHandle, id: u32) -> Result<User, String> {
+    let state: State<'_, AppState> = app_handle.state();
+
+    match state.user_manager().show(id).await {
+        Ok(user) => Ok(user),
+        Err(_) => Err("note not found".to_string()),
+    }
+}
```

Comment:
Should separate notes routes and user routes in their own separate files, this doesn't seem very scalable.